packages/packages/commonwell-cert-runner/src/document-consumption.ts

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env node
+import {
+  CommonWell,
+  convertPatiendIdToDocQuery,
+  getId,
+  getIdTrailingSlash,
+  isLOLA1,
+  RequestMetadata,
+} from "@metriport/commonwell-sdk";
+import {
+  getPatientStrongIds,
+  getPersonIdFromSearchByPatientDemo,
+} from "@metriport/commonwell-sdk/lib/common/util";
+import { docPatient, docPerson, metriportSystem } from "./payloads";
+
+// Document Consumption
+// https://commonwellalliance.sharepoint.com/sites/ServiceAdopter/SitePages/Document-Consumption-(SOAP,-REST).aspx
+
+// TODO Remove logs with the prefix '...'
+
+export async function documentConsumption(commonWell: CommonWell, queryMeta: RequestMetadata) {
+  // E1: Document Query
+  console.log(`>>> E1c: Query for documents using FHIR (REST)`);
+
+  console.log(`... Search for Patient...`, {
```

Comment:
Why the console log for search for patient?

Reply:
To be removed on the upcoming PR as indicated [here](https://github.com/metriport/metriport/pull/89/files/92021604b3ea944c165ca3283380de48f437672d#r1091161125). Kept to help debugging while I work on the next one.